include/mbedtls/config.h

```diff
@@ -256,6 +256,25 @@
  */
 //#define MBEDTLS_DEPRECATED_REMOVED
 
+/**
+ * \def MBEDTLS_CHECK_PARAMS
+ *
+ * This configuration controls whether the library validates parameters passed
```

Comment:
I think that `whether the library validates parameters passed to it.` is a bit vague. What kind of parameters are you talking about? Functions, globals, compile-time flags, etc...